ztp/policygenerator/policyGen/policyBuilderOverlay_test.go

```diff
@@ -74,6 +75,19 @@ spec:
 	assert.Equal(t, objDef["kind"], "JustForTest")
 	assert.NotNil(t, objDef["spec"])
 	assert.Nil(t, objDef["data"])
+	assert.NotNil(t, objDef["metadata"])
+
+	annotations := objDef["metadata"].(map[string]interface{})["annotations"].(map[string]interface{})
+	assert.NotNil(t, annotations)
+	assert.Equal(t, len(annotations), 2)
+	assert.Equal(t, annotations["key1"], "value1")
+	assert.Equal(t, annotations["key2"], "value2")
+
+	labels := objDef["metadata"].(map[string]interface{})["labels"].(map[string]interface{})
+	assert.NotNil(t, labels)
+	assert.Equal(t, len(labels), 2)
+	assert.Equal(t, annotations["key1"], "value1")
+	assert.Equal(t, annotations["key2"], "value2")
```

Comment:
Did you intend to validate the labels here rather than the annotations again?

Reply:
Good catch!! Thank you. Will update